projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-rectangular.php

```diff
@@ -207,7 +207,8 @@ public function get_grouped_images() {
 	public function apply_content_width( $width ) {
 		foreach ( $this->grouped_images as $row ) {
 			$row->width      = $width;
-			$row->raw_height = 1 / $row->ratio * ( $width - $this->margin * ( count( $row->groups ) - $row->weighted_ratio ) );
+			$group_count     = is_countable( $row->groups ) ? count( $row->groups ) : 0;
+			$row->raw_height = 1 / $row->ratio * ( $width - $this->margin * ( $group_count - $row->weighted_ratio ) );
```

Comment:
I wonder if there's a different way to handle this. It seems like having a `$group_count` of `0` is going to lead to unexpected results because the `$row->raw_height` will end up being negative.

Reply:
Having 
`$row->raw_height = 1 / $row->ratio * ( $width - $this->margin * max(0, $group_count - $row->weighted_ratio ) );`
would prevent the negative (in some cases), but it's hard to say if there would be unwanted consequences since it also depends on other variables like `$width`, `$this->margin`, and `$row->weighted_ratio`. It's a weird one 🤔

The proposed change doesn't change the existing behavior, though.